### ISSUE TITLE
Checkpresence does not accept MAC from an Iphone 12

### ISSUE THREAD

**Issue by jk-andersen** [afuerhoff/ioBroker.fb-checkpresence#55]
Hi There,

have a new iPhone 12 attached to my network. Adapter works fine with all other family members - After adding the iphone 12 it does not work with this device (Fixed MAC) - With this error:

```

[fb-checkpresence.0 | 2021-06-13 11:20:46.906 | warn | getActive Jan-neu: Warning: The configured mac-address for member Jan-neu was not found. Please insert a valid mac-address!
-- | -- | -- | --]
```

**Comment by afuerhoff** [comment]
Could you please send me the configuration values for this iPhone, the object values and the log with log level debug from the start of the adapter. Also the values for the iPhone in the Fritzbox are usefull (are there more than one entry for the same iphone).

**Comment by afuerhoff** [comment]
Hello, I have published a new version on github. Please test it.

**Comment by jk-andersen** [comment]
Hi,

I do not see a new version on github, yet.

Start Log:
`



2021-06-15 08:59:20.566 -&nbsp;info: host.iobroker "system.adapter.fb-checkpresence.0" disabled
--
2021-06-15 08:59:20.568 -&nbsp;info: host.iobroker stopInstance system.adapter.fb-checkpresence.0 (force=false, process=true)
2021-06-15 08:59:20.578 -&nbsp;info: host.iobroker stopInstance system.adapter.fb-checkpresence.0 send kill signal
2021-06-15 08:59:20.577 -&nbsp;info: fb-checkpresence.0 (609) Got terminate signal TERMINATE_YOURSELF
2021-06-15 08:59:20.579 -&nbsp;info: fb-checkpresence.0 (609) cleaned everything up ...
2021-06-15 08:59:21.079 -&nbsp;info: fb-checkpresence.0 (609) terminating
2021-06-15 08:59:21.080 -&nbsp;info: fb-checkpresence.0 (609) Terminated (ADAPTER_REQUESTED_TERMINATION): Without reason
2021-06-15 08:59:21.579 -&nbsp;info: host.iobroker stopInstance system.adapter.fb-checkpresence.0 killing pid 609
2021-06-15 08:59:21.605 -&nbsp;info: host.iobroker instance system.adapter.fb-checkpresence.0 terminated with code 11 (ADAPTER_REQUESTED_TERMINATION)
2021-06-15 08:59:25.349 -&nbsp;info: sonoff.0 (1177) State value to set for "sonoff.0.Tasmota4.Wifi_Channel" has to be stringified but received type "number"
2021-06-15 08:59:25.853 -&nbsp;info: sonoff.0 (1177) State value to set for "sonoff.0.Tasmota1.Wifi_Channel" has to be stringified but received type "number"
2021-06-15 08:59:26.312 -&nbsp;info: host.iobroker "system.adapter.fb-checkpresence.0" enabled
2021-06-15 08:59:26.362 -&nbsp;info: host.iobroker instance system.adapter.fb-checkpresence.0 started with pid 1848
2021-06-15 08:59:26.834 -&nbsp;info: fb-checkpresence.0 (1848) starting. Version 1.1.5 in /opt/iobroker/node_modules/iobroker.fb-checkpresence, node: v14.17.0, js-controller: 3.3.12
2021-06-15 08:59:27.163 -&nbsp;info: fb-checkpresence.0 (1848) start fb-checkpresence.0: FRITZ!Box 6591 Cable version: 161.07.27 ip-address: "192.168.178.1" - interval devices: 1 min. - interval members: 60 s
2021-06-15 08:59:27.164 -&nbsp;info: fb-checkpresence.0 (1848) configuration default connection: 1.WANIPConnection.1
2021-06-15 08:59:27.164 -&nbsp;info: fb-checkpresence.0 (1848) Hosts1-X_AVM-DE_GetHostListPath is supported
2021-06-15 08:59:27.164 -&nbsp;info: fb-checkpresence.0 (1848) Hosts1-X_AVM-DE_GetMeshListPath is supported
2021-06-15 08:59:27.164 -&nbsp;info: fb-checkpresence.0 (1848) Hosts1-GetSpecificHostEntry is supported
2021-06-15 08:59:27.164 -&nbsp;info: fb-checkpresence.0 (1848) Hosts1-X_AVM-DE_GetSpecificHostEntryByIP is supported
2021-06-15 08:59:27.165 -&nbsp;info: fb-checkpresence.0 (1848) DeviceInfo1-GetSecurityPort is supported
2021-06-15 08:59:27.165 -&nbsp;info: fb-checkpresence.0 (1848) WANCommonInterfaceConfig1-GetCommonLinkProperties is supported
2021-06-15 08:59:27.165 -&nbsp;info: fb-checkpresence.0 (1848) WANPPPConnection1-GetInfo is supported
2021-06-15 08:59:27.165 -&nbsp;info: fb-checkpresence.0 (1848) WANIPConnection1-GetInfo is supported
2021-06-15 08:59:27.165 -&nbsp;info: fb-checkpresence.0 (1848) WLANConfiguration3-SetEnable is supported
2021-06-15 08:59:27.165 -&nbsp;info: fb-checkpresence.0 (1848) WLANConfiguration3-GetInfo is supported
2021-06-15 08:59:27.165 -&nbsp;info: fb-checkpresence.0 (1848) WLANConfiguration3-GetSecurityKeys is supported
2021-06-15 08:59:27.165 -&nbsp;info: fb-checkpresence.0 (1848) DeviceInfo1-GetInfo is supported
2021-06-15 08:59:27.165 -&nbsp;info: fb-checkpresence.0 (1848) X_AVM-DE_HostFilter-DisallowWANAccessByIP is supported
2021-06-15 08:59:27.165 -&nbsp;info: fb-checkpresence.0 (1848) X_AVM-DE_HostFilter-GetWANAccessByIP is supported
2021-06-15 08:59:27.166 -&nbsp;info: fb-checkpresence.0 (1848) DeviceConfig1-Reboot is supported
2021-06-15 08:59:27.166 -&nbsp;info: fb-checkpresence.0 (1848) WANPPPConnection1-ForceTermination is supported
2021-06-15 08:59:27.166 -&nbsp;info: fb-checkpresence.0 (1848) LANConfigSecurity1-X_AVM-DE_GetCurrentUser is supported
2021-06-15 08:59:27.166 -&nbsp;info: fb-checkpresence.0 (1848) Layer3Forwarding1-GetDefaultConnectionService is supported
2021-06-15 08:59:27.246 -&nbsp;info: fb-checkpresence.0 (1848) createGlobalObjects finished successfully
2021-06-15 08:59:27.432 -&nbsp;info: fb-checkpresence.0 (1848) createMemberObjects finished successfully
2021-06-15 08:59:28.757 -&nbsp;info: fb-checkpresence.0 (1848) createFbDeviceObjects finished successfully
2021-06-15 08:59:28.757 -&nbsp;info: fb-checkpresence.0 (1848) states successfully subscribed
2021-06-15 08:59:28.757 -&nbsp;info: fb-checkpresence.0 (1848) loop successfully started
2021-06-15 08:59:29.910 -&nbsp;warn: fb-checkpresence.0 (1848) getActive Jan-neu: Warning: The configured mac-address for member Jan-neu was not found. Please insert a valid mac-address!



`
And config - The other phone works fine
![1](https://user-images.githubusercontent.com/62281415/122007394-2e560d00-cdb8-11eb-9502-67f841bfa35e.PNG)

**Comment by jk-andersen** [comment]
Und hier noch die Fritzbox - Gerät ist nicht doppelt da.
![2](https://user-images.githubusercontent.com/62281415/122014458-74629f00-cdbf-11eb-89fd-6d267da045c3.PNG)

**Comment by afuerhoff** [comment]
The new version is only a quick test version with the same version label 1.1.4. You have to install it over the github cat in iobroker.

**Comment by jk-andersen** [comment]
Installed - Now both devices dont work :)

`



2021-06-15 12:17:30.490 -&nbsp;info: fb-checkpresence.0 (3356) starting. Version 1.1.5 in /opt/iobroker/node_modules/iobroker.fb-checkpresence, node: v14.17.0, js-controller: 3.3.12
--
2021-06-15 12:17:30.815 -&nbsp;info: fb-checkpresence.0 (3356) start fb-checkpresence.0: FRITZ!Box 6591 Cable version: 161.07.27 ip-address: "192.168.178.1" - interval devices: 1 min. - interval members: 60 s
2021-06-15 12:17:30.816 -&nbsp;info: fb-checkpresence.0 (3356) configuration default connection: 1.WANIPConnection.1
2021-06-15 12:17:30.816 -&nbsp;info: fb-checkpresence.0 (3356) Hosts1-X_AVM-DE_GetHostListPath is supported
2021-06-15 12:17:30.816 -&nbsp;info: fb-checkpresence.0 (3356) Hosts1-X_AVM-DE_GetMeshListPath is supported
2021-06-15 12:17:30.817 -&nbsp;info: fb-checkpresence.0 (3356) Hosts1-GetSpecificHostEntry is supported
2021-06-15 12:17:30.817 -&nbsp;info: fb-checkpresence.0 (3356) Hosts1-X_AVM-DE_GetSpecificHostEntryByIP is supported
2021-06-15 12:17:30.817 -&nbsp;info: fb-checkpresence.0 (3356) DeviceInfo1-GetSecurityPort is supported
2021-06-15 12:17:30.817 -&nbsp;info: fb-checkpresence.0 (3356) WANCommonInterfaceConfig1-GetCommonLinkProperties is supported
2021-06-15 12:17:30.817 -&nbsp;info: fb-checkpresence.0 (3356) WANPPPConnection1-GetInfo is supported
2021-06-15 12:17:30.817 -&nbsp;info: fb-checkpresence.0 (3356) WANIPConnection1-GetInfo is supported
2021-06-15 12:17:30.817 -&nbsp;info: fb-checkpresence.0 (3356) WLANConfiguration3-SetEnable is supported
2021-06-15 12:17:30.817 -&nbsp;info: fb-checkpresence.0 (3356) WLANConfiguration3-GetInfo is supported
2021-06-15 12:17:30.817 -&nbsp;info: fb-checkpresence.0 (3356) WLANConfiguration3-GetSecurityKeys is supported
2021-06-15 12:17:30.818 -&nbsp;info: fb-checkpresence.0 (3356) DeviceInfo1-GetInfo is supported
2021-06-15 12:17:30.818 -&nbsp;info: fb-checkpresence.0 (3356) X_AVM-DE_HostFilter-DisallowWANAccessByIP is supported
2021-06-15 12:17:30.818 -&nbsp;info: fb-checkpresence.0 (3356) X_AVM-DE_HostFilter-GetWANAccessByIP is supported
2021-06-15 12:17:30.818 -&nbsp;info: fb-checkpresence.0 (3356) DeviceConfig1-Reboot is supported
2021-06-15 12:17:30.818 -&nbsp;info: fb-checkpresence.0 (3356) WANPPPConnection1-ForceTermination is supported
2021-06-15 12:17:30.818 -&nbsp;info: fb-checkpresence.0 (3356) LANConfigSecurity1-X_AVM-DE_GetCurrentUser is supported
2021-06-15 12:17:30.818 -&nbsp;info: fb-checkpresence.0 (3356) Layer3Forwarding1-GetDefaultConnectionService is supported
2021-06-15 12:17:30.887 -&nbsp;info: fb-checkpresence.0 (3356) createGlobalObjects finished successfully
2021-06-15 12:17:31.007 -&nbsp;info: fb-checkpresence.0 (3356) createMemberObjects finished successfully
2021-06-15 12:17:32.163 -&nbsp;info: fb-checkpresence.0 (3356) createFbDeviceObjects finished successfully
2021-06-15 12:17:32.163 -&nbsp;info: fb-checkpresence.0 (3356) states successfully subscribed
2021-06-15 12:17:32.164 -&nbsp;info: fb-checkpresence.0 (3356) loop successfully started
2021-06-15 12:17:33.319 -&nbsp;warn: fb-checkpresence.0 (3356) getActive Rike: TypeError: Cannot read property 'includes' of undefined
2021-06-15 12:17:33.320 -&nbsp;warn: fb-checkpresence.0 (3356) getActive Jan-neu: TypeError: Cannot read property 'includes' of undefined



`

**Comment by afuerhoff** [comment]
Which node.js version do you have installed?

**Comment by jk-andersen** [comment]
v14.17.0

**Comment by afuerhoff** [comment]
Hello, I have published a new version on github. I hope this works. I press my thumbs. 
Please test it.

**Comment by jk-andersen** [comment]
Installed it - sadly no change - Different error, though.


fb-checkpresence.0 | 2021-06-16 09:41:14.294 | warn | getActive Jan-neu: TypeError: Cannot read property 'toString' of undefined
-- | -- | -- | --
fb-checkpresence.0 | 2021-06-16 09:41:14.293 | warn | getActive Rike: TypeError: Cannot read property 'toString' of undefined

**Comment by afuerhoff** [comment]
Hello, I hope I have found the problem now. Please test again.

**Comment by jk-andersen** [comment]
Yes - It works!

Error Messages are gone, new Device is accepted.
I had to delete all family-members in the objects-tree first, now it recognizes the changes.
Thanks a lot!



fb-checkpresence.0 | 2021-06-17 08:53:05.306 | info | newActive Jan-neu true
-- | -- | -- | --
fb-checkpresence.0 | 2021-06-17 08:53:04.132 | info | loop successfully started

**Comment by afuerhoff** [comment]
It is fixed with version v1.1.7. The version is in the latest repository now.
